doc/rbd/rbd-snapshot.rst

```diff
@@ -66,75 +73,94 @@ Create Snapshot
 ---------------
 
 To create a snapshot with ``rbd``, specify the ``snap create`` option, the pool
-name and the image name.  ::
+name, and the image name:
 
-	rbd snap create {pool-name}/{image-name}@{snap-name}
+.. prompt:: bash $
 
-For example:: 
+   rbd snap create {pool-name}/{image-name}@{snap-name}
 
-	rbd snap create rbd/foo@snapname
+For example:
+
+.. prompt:: bash $
+
+   rbd snap create rbd/foo@snapname
 	
 
 List Snapshots
 --------------
 
-To list snapshots of an image, specify the pool name and the image name. ::
+To list the snapshots of an image, specify the pool name and the image name:
```

Comment:
Ditto -- "To list the snapshots of an image, use `rbd snap ls` command and specify ...".

Reply:
Done.